src/google/cloud/ndb/model.py

```diff
@@ -3973,6 +3973,83 @@ def _put_async(self, **options):
 
     put_async = _put_async
 
+    @classmethod
+    def _query(
+        cls,
+        *filters,
+        distinct=False,
+        ancestor=None,
+        order_by=None,
+        orders=None,
+        project=None,
+        app=None,
+        namespace=None,
+        projection=None,
+        distinct_on=None,
+        group_by=None,
+    ):
+        """Generate a query for this class.
+
+        Args:
+            *filters (query.FilterNode): Filters to apply to this query.
+            distinct (Optional[bool]): Setting this to :data:`True` is
+                shorthand for setting `distinct_on` to `projection`.
+            ancestor (key.Key): Entities returned will be descendants of
+                `ancestor`.
+            order_by (list[Union[str, google.cloud.ndb.model.Property]]):
```

Comment:
Using ``list`` here, instead of ``Sequence``, because for some reason the Sphinx Napoleon plugin is choking on ``Sequence``, despite it being a member of ``typing`` in the standard library.

Reply:
Huh, sphinx seems to be giving us one sort of trouble or another recently.